tmessage/cli.py

```diff
@@ -36,6 +37,12 @@ def on_message(client, userdata, message):
         print(Back.GREEN + Fore.BLACK + current_msg +
               Back.RESET + Fore.RESET + "")
 
+def store_messages(raw_msg):
+    f = open('messages/message.txt', 'a')

```

Comment:
@Shanmugapriya03 You have to first check if `messages` directory exists or not. Otherwise, it will give this:
`FileNotFoundError: [Errno 2] No such file or directory: 'messages/message.txt'` which means that you have to create the directory first.

Reply:
Ya, But then I just added the messages folder default and proceeded. So it will not throw an error. Do you still want me to make changes as you said?

Reply:
You can use `os` module along with `try` and `except` for that, `os.mkdir('messages')`.

Reply:
Yes, do the changes please. It will not be good to show the error when we can create a new directory.